api/src/pcapi/core/offerers/repository.py

```diff
@@ -280,6 +281,18 @@ def has_digital_venue_with_at_least_one_offer(offerer_id: int) -> bool:
 )
 
 
+def get_by_collective_booking_id(booking_id: int) -> models.Offerer:
```

Comment:
(détail / nommage) Puisque `core.offerers` inclut des fonctions relatives aux structures, mais aussi aux lieux et à d'autres modèles, on pourrait préciser le nom de cette fonction : 

```suggestion
def get_offerer_by_collective_booking_id(booking_id: int) -> models.Offerer:
```